src/main/java/com/mentoring/simplewebservice/rest/UserResource.java

```diff
@@ -0,0 +1,86 @@
+package com.mentoring.simplewebservice.rest;
+
+import com.mentoring.simplewebservice.domain.User;
+import com.mentoring.simplewebservice.service.UserService;
+import com.sun.jersey.multipart.FormDataParam;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.io.*;
+
+@Path("api/v1/user")
```

Comment:
I'd recommend to have /users in the resource path.
Also please use userId as key instead of login. This is the common approach for REST API building